src/docs/patterns/visualizations.mdx

```diff
@@ -1,62 +1,93 @@
-import { DemoCard, ImageGrid, MaterialDesignDescription } from '../../app/components';
+import { DemoCard, ImageGrid, MaterialDesignDescription, Divider } from '../../app/components';
 
 import VisualizationBanner from '../../app/assets/design-patterns/visualizations/visualizations-banner.svg';
+import LineChart from '../../app/assets/design-patterns/visualizations/line-chart.png';
+import PieChart from '../../app/assets/design-patterns/visualizations/pie-chart.png';
+import ClockwisePie from '../../app/assets/design-patterns/visualizations/cw-ccw.png';
+import MapsImg from '../../app/assets/design-patterns/visualizations/map.png';
+import DontUsePrimaryColor from '../../app/assets/design-patterns/visualizations/use-primary-color.png';
 
 <!-- KEYWORDS: design pattern visualization map chart highcharts chartjs mapbox -->
 
 # Visualizations
 
 <ImageGrid regularWidth images={[VisualizationBanner]} />
 
-There are many opportunities to visualize data in different applications. The most common visualizations used in PX Blue applications come in the form of charts/graphs and maps.
+There are many opportunities to visualize data in different applications. The most common visualizations used in PX Blue applications come in the form of charts / graphs and maps.
+
+<Divider />
+
+# Charts / Graphs
+
+<ImageGrid
+    images={[LineChart, PieChart]}
+    caption={'Line charts (#1) and pie / ring charts (#2) are the most frequently used data visualizations in PX Blue.'}
+/>
+
+There are [many variations](https://material.io/design/communication/data-visualization.html#types) of data visualizations. They can be a big line chart with 8 series, occupying the entire screen; or they can be as tiny as a simple pie chart embedded in [lists](/patterns/lists).
 
 <MaterialDesignDescription
     title={'Data Visualization'}
     url={'https://material.io/design/communication/data-visualization.html'}
 />
 
-## Charts & Graphs (Highcharts & ChartJS)
-
-Currently, PX Blue supports building charts and graphs using [Highcharts](http://www.highcharts.com) and [ChartJS](https://www.chartjs.org/), which are comprehensive libraries compatible with both Angular and React.
-
-> **NOTE:** the following examples merely serve to show how to bring the charting libraries into your application and adjust some of the common chart parameters. You will want to adjust these values to meet the design requirements of your application.
-
-### Highcharts
+> The following examples merely serve to show how to bring the charting libraries into your application and adjust some of the common chart parameters. You will want to adjust these values to meet the design requirements of your application.
```

Comment:
Can we get rid of this and add a story to the backlog to revisit the design of these examples to follow our recommendations?

Reply:
PX Bleu 1613